app/src/main/res/drawable/edit_text_background_border.xml

```diff
@@ -5,5 +5,5 @@
   <solid android:color="@color/white" />
   <stroke
     android:width="1dp"
-    android:color="@color/grey" />
+    android:color="#666666" />
```

Comment:
Introduce this in colors file
`<color name="oppiaGreyBorderDark">#666666</color>`

Reply:
okay